src/index.ts

```diff
@@ -73,6 +81,18 @@ const convertStack = (stack: Stack, options: Options = {}) => {
     });
   }
 
+  if (ignoreCurrentVersion && template.Parameters && template.Resources) {
+    template.Parameters = expect.any(Object);
+    for (const [key, resource] of Object.entries(template.Resources)) {
+      const match = currentVersionRegex.exec(key);
```

Comment:
What should we do with Refs? For example:
```typescript
new CfnOutput(stack, "CurrentVersion", {
    value: lf.currentVersion.functionArn,
  });
```
==>
```
    +   "Outputs": Object {
    +     "CurrentVersion": Object {
    +       "Value": Object {
    +         "Ref": "FunctionCurrentVersion4E2B22619c30a1bedea6725a7f3564ff33a6d535",
    +       },
    +     },
    +   },
```

Reply:
Would something like this do the trick?

```
// Sets references to null where they match the regex
const nullifyReferences = (
  template: unknown,
  regex: RegExp = currentVersionRegex
) => {
  // Nullify references from an array
  if (Array.isArray(template)) {
    for (let i = 0; i < template.length; i++) {
      if (typeof template[i] === 'string') {
        const match = regex.exec(template[i]);
        if (match) { 
          template[i] = null;
        } 
      } else {
        nullifyReferences(template[i], regex);
      }
    }
  }

  // Nullify references from an object
  if (
    typeof template === 'object' &&
    !Array.isArray(template) &&
    template !== null
  ) {
    for (const key of Object.keys(template)) {
      const match = currentVersionRegex.exec(key);
      if (match) {
        template[key] = null;
      } else {
        nullifyReferences(template[key], regex);
      }
    }
  }

  // Nullify strings
  if (typeof template === 'string') {
    const match = regex.exec(template);
    if (match) { 
      template = null;
    } 
  }
}

...

  if (ignoreCurrentVersion && template.Parameters && template.Resources) {
    template.Parameters = expect.any(Object);
    for (const [key, resource] of Object.entries(template.Resources)) {
      const match = currentVersionRegex.exec(key);
      if (match) {
        const newKey = `${match[1]}xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx`;
        template.Resources[newKey] = resource;
        delete template.Resources[key];
      } else {
        nullifyReferences(resource);
      }
    }
  }
```

Reply:
You can try it out very easily. Simply create a test case with an output parameter
```typescript
new CfnOutput(stack, "CurrentVersion", {
    value: lf.currentVersion.functionArn,
  });
```